modules/datadog-role/main.tf

```diff
@@ -0,0 +1,98 @@
+# This module was updated using terraform 0.11.4 at 2018/04/09.
+terraform {
+  required_version = ">= 0.11.4"
+}
+
+locals {
+  datadog_account_id = "464622532012"
```

Comment:
Are we going to use the same datadog account id for all products and all environment (prod & staging)?